recipes/libuv/all/patches/1.38.0/fix-cmake.patch

```diff
@@ -0,0 +1,78 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
libuv's CMake code underwent a large cleaning in v1.35.0.  This patch appears to be a correct conversion of what 1.34.2's `fix-cmake.patch` did, but I would appreciate extra scrutiny on it.